tools/install.sh

```diff
@@ -48,6 +48,24 @@ case $os in
       echo
       echo "Please visit https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Installing-OpenBangla-Keyboard for Distrowise/Distro-specific Install Instructions."
       echo
-      ;;
+      
+      echo "Do you want to see the instructions for installing OpenBangla Keyboard in your system?"
+      select yn in "Yes" "No"; do
+      
+      case $yn in
+            
+            (Yes)
```

Comment:
Have you tested this script? The answer [here](https://stackoverflow.com/questions/226703/how-do-i-prompt-for-yes-no-cancel-input-in-a-linux-shell-script#226724) uses a trailing `)` if you look closely. ShellCheck accepts your code though!

Reply:
I am not saying that your syntax is wrong here, because I am not sure about that. I didn't know about this feature before! Thus I am asking if you tested running the script.

Reply:
I can not see the trailing `)`. It does not require a wildcard as there is only two option. Yes or No

Reply:
I was talking about the second snippet  :
```
echo "Do you wish to install this program?"
select yn in "Yes" "No"; do
    case $yn in
        Yes ) make install; break;;
        No ) exit;;
    esac
done
```
 :relaxed: I think your syntax might be right also.